app/globals.css

```diff
@@ -65,3 +65,8 @@
     --ring: 212.7 26.8% 83.9%;
   }
 }
+
+.LeagueCardEliminated h4,
+.LeagueCardEliminated * {
+  @apply dark:text-zinc-50;
+}
```

Comment:
Q: Hey Ryan! Curious as to why this is indicated here in globals.css? I'd assume `components/LeagueCard.tsx` could take care of this?

Reply:
I think I figured it out... will update the PR accordingly.

Reply:
@vmaineng I've rewritten the page and components so now this is no longer needed. Global CSS and the associated components have been updated :)

Reply:
Updated in latest commit.